tools/integration/lib/harvester.js

```diff
@@ -0,0 +1,80 @@
+// (c) Copyright 2024, SAP SE and ClearlyDefined contributors. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const { callFetch, buildPostOpts } = require('./fetch')
+
+const defaultToolChecks = [
+  ['licensee', '9.14.0'],
+  ['scancode', '30.3.0'],
+  ['reuse', '3.2.1']
+]
```

Comment:
What I find slightly confusing here is that this uses the schema version not the tool version. Not sure how to make this clearer and avoid confusion for future maintainers though 😅

Reply:
I can also add some comments here and testConfig.js to clarify this.

Reply:
Will address this as a separate PR.